test/parallel/test-whatwg-url-domainto.js

```diff
@@ -35,6 +36,13 @@ const tests = require('../fixtures/url-idna.js');
 }
 
 {
+  for (const [i, { ascii, unicode }] of testsHyphenDomains.valid.entries()) {
+    assert.strictEqual(ascii, domainToASCII(unicode),
```

Comment:
If it's only for testing those domains won't get converted, maybe just make the test cases an array of string and `assert.strictEqual(domain, domainToASCII(domain))`?